hadoop-plugin/src/test/groovy/com/linkedin/gradle/azkaban/AzkabanHelperTest.groovy

```diff
@@ -16,10 +16,16 @@
 package com.linkedin.gradle.azkaban;
 
 import org.gradle.api.GradleException;
-import org.json.JSONObject;
-import org.junit.Test;
+import org.json.JSONObject
+import org.junit.Rule;
+import org.junit.Test
+import org.junit.rules.ExpectedException
+import org.junit.runner.RunWith;
+
 import static org.junit.Assert.assertEquals;
+//import org.powermock.modules.junit4.PowerMockRunner;
```

Comment:
If these aren't needed, could be get rid of them? Is there a reason to keep commented-out code around?

Reply:
Removed.